typescript/build-model.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+set -euo pipefail
+
+cp ../lib/typescript/model.ts .
```

Comment:
I think this script should run from `${CMAKE_BINARY_DIR}`, `rm -rf model.ts-package`, recreate `model.ts-package`, enter, do its stuff, leave the archive in `${CMAKE_BINARY_DIR}`.

Reply:
Done